src/index.ts

```diff
@@ -100,7 +105,8 @@ export const crud = <M extends Model>(
 
 const getList = <M extends Model>(
   model: { new (): M } & typeof Model,
-  hooks?: GetListHooks
```

Comment:
Why did you switch to `Something | undefined`?

Reply:
because the param is always passed but sometime it's undefined